app/components/ledger/CredentialInfoLedgerDetails.tsx

```diff
@@ -11,13 +13,17 @@ const CredentialInfoLedgerDetails = (props: Props) => (
     <div className="textLeft">
         <p className="mT0">Please confirm details on ledger:</p>
         <p>
-            <b>Public key:</b> {props.credentialPublicKeys.keys[0].verifyKey}
+            <b>Public key:</b>
+            <PublicKeyDetails
+                publicKey={props.credentialPublicKeys.keys[0].verifyKey}
+            />
         </p>
         <p>
             <b>Signature threshold:</b> {props.credentialPublicKeys.threshold}
         </p>
         <p>
-            <b>Registration ID credential (RegIdCred):</b> {props.credId}
+            <b>Registration ID credential (RegIdCred):</b>
+            <PublicKeyDetails publicKey={props.credId} />
```

Comment:
It is not a public key you are supplying here. Perhaps it would be better to make `PublicKeyDetails` into a more generic component for displaying hex strings, and then make `PublicKeyDetails` use that.